sup3r/preprocessing/data_handling.py

```diff
@@ -76,12 +77,199 @@ def get_handler_class(file_paths):
         HandlerClass = DataHandlerH5
         if all('nsrdb' in os.path.basename(fp) for fp in file_paths):
             HandlerClass = DataHandlerH5SolarCC
+    elif not is_time_series(file_paths):
+        HandlerClass = DataHandlerNCforCC
     else:
         HandlerClass = DataHandlerNC
     return HandlerClass
 
 
-class DataHandler(FeatureHandler):
+class InputHandler:
```

Comment:
I think this is commonly called a [Mixin class](https://www.thedigitalcatonline.com/blog/2020/03/27/mixin-classes-in-python/#:~:text=Mixin%20classes%20in%20Python%C2%B6&text=In%20Python%2C%20so%2Dcalled%20mixins,with%20the%20other%20parent%20classes.), you should refer to it / name it as such